tests/phpunit/includes/testcase-dominant-color.php

```diff
@@ -0,0 +1,140 @@
+<?php
+require_once __DIR__ . '/../tests/image/base.php';
```

Comment:
Do we need to bootstrap this?

Reply:
yes is error without it
I could move it to the general includes but that feels wrong